src/components/SelectAssets/SelectAssetView.tsx

```diff
@@ -9,8 +9,8 @@ import {
   useLocation
 } from 'react-router-dom'
 import { AccountSpecifier } from 'state/slices/portfolioSlice/portfolioSlice'
-import { selectPortfolioAccounts } from 'state/slices/portfolioSlice/selectors'
 import { findAccountsByAssetId } from 'state/slices/portfolioSlice/utils'
```

Comment:
nup, let's keep this PR completely to file structure, we can address this in another PR.

Reply:
I think `selectPortfolioAccounts` selector was used intentionally, getting all accounts and filtering them whenever the asset is selected. the thing about `selectAccountIdsByAssetId` is, it needs the `assetId` param to work, but in this component, `assetId` could be null (if you click the back button on modal which goes to `AssetSearch` component) which can cause a lot of side-effects if changed to `selectAccountIdsByAssetId `.
also somehow assetId in location state is not in sync with `ReceiveRouter`'s `selectedAsset`. 
But I can fix them in another branch since I think that's out of the scope of this PR.